code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -0,0 +1,55 @@
+/obj/structure/closet/cardboard
+	name = "large cardboard box"
+	desc = "Just a box..."
+	icon_state = "cardboard"
+	health = 10 //At the end of the day it's still just a box
+	mob_storage_capacity = 1
+	burntime = 20
+	can_weld_shut = 0
+	cutting_tool = /obj/item/weapon/wirecutters
+	open_sound = 'sound/effects/rustle2.ogg'
+	cutting_sound = 'sound/items/poster_ripped.ogg'
+	material_drop = /obj/item/stack/sheet/cardboard
+	var/move_delay = 0
+	var/egg_cooldown = 0
+
+/obj/structure/closet/cardboard/relaymove(mob/user, direction)
+	if(opened || move_delay || user.stat || user.stunned || user.weakened || user.paralysis || !isturf(loc) || !has_gravity(loc)/*|| Kawanishi-Noseguchi || Kinunobebashi || Takiyama || Uguisunomori || Tsuzumigataki || Tada || Hirano || Ichinotorii || Uneno || Yamashita || Sasabe || Kofudai || Tokiwadai || Myokenguchi*/)
+		return
+	step(src, direction)
+	move_delay = 1
+	spawn(config.walk_speed) //Kept you waiting, huh?
+		move_delay = 0
+
+/obj/structure/closet/cardboard/open() //!
+	if(opened || !can_open())
+		return 0
+	if(!egg_cooldown)
+		var/mob/living/Snake = null
+		for(var/mob/living/L in src.contents)
+			Snake = L
+			break
+		if(Snake)
+			var/list/alerted = viewers(7,src)
+			if(alerted)
+				for(var/mob/living/L in alerted)
+					if(!L.stat)
+						L.do_alert_animation(L)
+				alerted << sound('sound/machines/chime.ogg') //HQ HQ!
+				egg_cooldown = 1
+				spawn(3000)
+					egg_cooldown = 0
```

Comment:
Shouldn't all this egg cooldown be something along the lines of 

if(egg_cooldown < world.time)

And then egg_cooldown = world.time + 3000

Instead of using spawn?